fast64_internal/f3d/f3d_writer.py

```diff
@@ -407,13 +410,7 @@ def addCullCommand(obj, fMesh, transformMatrix, matWriteMethod):
     # if the object has a specifically set culling bounds, use that instead
     for vertexPos in obj.get("culling_bounds", obj.bound_box):
         fMesh.cullVertexList.vertices.append(
-            F3DVert(
-                Vector(vertexPos),
-                [0, 0],
-                Vector([0, 0, 0]),
-                None,
-                0.0,
-            ).toVtx(
+            F3DVert(Vector(vertexPos), [0, 0], Vector([0, 0, 0]), None, 0.0,).toVtx(
```

Comment:
```suggestion
            F3DVert(
                Vector(vertexPos),
                [0, 0],
                Vector([0, 0, 0]),
                None,
                0.0,
            ).toVtx(
```
Fix the formatting here